salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc

```diff
@@ -30,13 +30,13 @@ This Mule flow shows how to upsert data via the streaming API to an object's end
 Here is what each operation is doing:
 
 * *HTTP Listener* +
-Accepts data from HTTP PUT requests with the connector and object name as the URI parameters.
+Accepts data from HTTP PUT requests with the source and object name as the URI parameters.
```

Comment:
@matthew-zizheng-li I think this should `HTTP POST`

Reply:
The HTTP trigger is doing PUT => POST

Do we want to keep the protocol POST all the way through?

Reply:
I guess this is a bigger philosophy question...